.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
   - 8
-  - 6
+  - 7
```

Comment:
We should add a `.nvmrc` file with at least version **7** because the tests are not working with nodejs versions older than 7.

Reply:
I agree. It's because of my use of async/await which node only supports in node 7.2 or greater

Reply:
I've never used NVM and I find their [documentation](https://github.com/creationix/nvm#nvmrc) lacking, do you know how I would do this?

Reply:
Normaly it should be enough to do something like this:
```bash
$ echo "7" > .nvmrc
$ nvm use
Found '/home/webmatze/Workplace/webmatze.github.com/hyperborder/.nvmrc' with version <7>
Now using node v7.10.1 (npm v4.2.0)
```
This should automatically select the latest version 7 release of nodejs that has been installed on your system.

Reply:
Makes sense, but we want `7.2` as the minimum, and version 8 is acceptable also.

Reply:
Yeah, that's right. But you are not forced to use the version. It is only a guide for future developers. Sadly we can not use something like '>=7.2' in `.nvmrc`.

Reply:
Done =]